osu.Game/Screens/Select/BeatmapInfoWedgeV2.cs

```diff
@@ -183,6 +185,14 @@ void removeOldInfo()
 
                     removeOldInfo();
                     Add(DisplayedContent = loaded);
+
+                    Info.StarRatingDisplay.DisplayedStars.BindValueChanged(s =>
```

Comment:
Rather than exposing this whole thing, expose just the bindable.

Inside `WedgeInfoText`:

```public Bindable<whatever> DisplayedStars => Info.StarRatingDisplay.DisplayedStars`

Reply:
right, thanks ,will do this going forward for similar cases